python/ray/autoscaler/_private/commands.py

```diff
@@ -737,6 +718,54 @@ def get_or_create_head_node(config: Dict[str, Any],
         cli_logger.print("  {}", remote_shell_str.strip())
 
 
+def _set_up_config_for_head_node(config: Dict[str, Any],
+                                 provider: NodeProvider,
+                                 no_restart: bool) ->\
+        Tuple[Dict[str, Any], Any]:
```

Comment:
Isn't this:
`Tuple[Dict[str, Any], Dict[str, Any]]`?
Can we define a global variable `ClusterConfig = Dict[str, Any]` and use it in the 3 places?

Reply:
The second return item is a named temporary file (note doc string). Unfortunately, it's hard to type those due to the structure of the `tempfile` module (https://stackoverflow.com/questions/64429113/how-should-a-namedtemporaryfile-be-annotated)

I think `ClusterConfig = Dict[str, Any]` would be a good thing to define globally somewhere and then use all over the code base. For now, won't do it for these two instances.